sfcasts/command_intro.md

```diff
@@ -0,0 +1,93 @@
+# Pattern Definition
+In this episode we are going to talk about the command pattern, which is a behavioral pattern.
+If you don't remember what it means, what are you doing here? Go back to episode 1! Just kidding.
+Here's a quick reminder. Behavioral patterns are those help you design classes with specific 
+responsibilities that can then work together, instead of putting all of that code into one giant class.
+
+Official definition: The command pattern encapsulates a request as a stand-alone object,
+allowing parameterization of clients with different requests, queueing or logging requests,
+and support undoable operations.
+Ok, I'm more confused now! Let's try it again.
+
+A more friendly definition: The command pattern encapsulates a task as a stand-alone object,
+decoupling what it does, how it does it, and when it gets done. Also, it allows to undo its actions
+by remembering enough information of what things need to be reverted.
+Ok, I think it makes more sense now.
+
+# Pattern Anatomy
+Let's see the parts that compose the command pattern:
+- It has a Command Interface with a single public method that's usually called `execute()`, it can
```

Comment:
Having big spots of talking and no coding is common in the early chapters - and probably even more common with the topics you like to cover Diego ;) - and we can use Danielle to add diagrams, which look great, but that's a slow process. So we need to keep in mind: what's being shown on the screen here? The ideal / laziest is to show some thing - code in editor, website, etc - so that we have to add minimal diagrams after. Long video pauses are the enemy, that's why I eventually learned to not write scripts in this much detail. But then talk while video recording then transcribe THAT into this rough script. That help with timing and avoiding too much talking while nothing happens on your computer. 

Video recording is 95% just coding and saying out loud what you're doing. And 5% explanations that form a basis for the script

Reply:
Thank you for the insight. I'll keep it in mind. I thought it would be easier for me if I just read my script while recording because I don't want to improvise the explanation, but well, that's just a matter of practice and confidence